v22.1/security-reference/transport-layer-security.md

```diff
@@ -0,0 +1,280 @@
+
+This is a complex social and technological problem, and the primary solution used today by the internet as a whole, as well as organizations across the world, is mix of careful practices, social relationships, and digital technologies. It is what is known as Public Key Infrastructure (PKI).
+
+At its core, PKI is a hierarchy of cryptographically backed trust relationships between the following categories of interested party:
+
+- **Subscribers** wish to use the PKI to prove their identity to the world and offer secure services to others.
+- **Relying parties** wish to connect with subscribers, secure in the their knownledge of the identity of the latter.
+- **Certificate Authorities (CA)** are responsible for verifying the identity of subscribers (which can include subornidate certificate authorties).
+
+### Certificates, signing, trust and authority
+
+The core mechanism of PKI is a the PKI certificate, also known as a 'security certificate', 'digital certificate' or 'TLS certificate' (because it is used in TLS), or abbreviated "cert".
+
+A PKI certificate is a file containing the following:
+
+- A) A public key to be used for [key pair encryption, often including TLS](#what-is-transport-layer-security-tls).
+- B) Some metadata about:
+	- the party that allegedly holds the corresponding private key and who therefore is the only one capable of decrypting messages encrypted with **A**, most importantly at least one **name**, such as a domain name in a domain name registry system (DNS). The certifiate essentially functions as a badge or nametage, allowing the holder to claim to be the named party.
+	- the party signing the certificate, and the certificate authority (if any) that signed its public certificate
+- C) A list of actions the holder of the certificate is thereby authorized to perform.
+
+On its own, such a digital certificate is of no more value than a paper certificate. Perhaps less value, as it can be neither scribbled upon nor burned. However, digital certificates have the advantage that they can be cryptographically **signed**, again using the mechanism of a public/private key pair.
+
+**Signing** here means encrypting the certificate using a private key, so that it can be decrypted with a public key. So how does that help? This is the point where the boundaries between computing systems and social systems become very murky. By signing a certificate (and anyone with a private key can do this) a party is acting as a "certificate authority"; they are in effect asserting the validity of the identity claim being made by the certificate holder.
+
+The premise of PKI is that if I present you with a certificate, and you can decrypt it with, for example, Cockroach Labs' public certificate, and the decrypted version says that I work for the Cockroach Labs documentation team, essentially you may interpret that Cockroach Labs offers its guarantee that I work for the documentation team (or at least, I did when the certificate was signed).
```

Comment:
```suggestion
The premise of PKI is that if I present you with a certificate, and you can decrypt it with, for example, a public certificate owned by Cockroach Labs, and the decrypted version says that I work for the Cockroach Labs documentation team, essentially you may interpret that Cockroach Labs offers its guarantee that I work for the documentation team (or at least, I did when the certificate was signed).
```
I think this is a bit too personalized. Maybe stick with `example.com` type things.